docs/monitoring-usage.rst

```diff
@@ -278,3 +278,86 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation for the `Creating Custom Metrics`_ for more information.
+
+To create a custom :class:`~gcloud.monitoring.timeseries.TimeSeries` value, you must first create a
+custom :class:`~gcloud.monitoring.metric.MetricDescriptor` object, and then use its
+type to create a fully parameterized :class:`~gcloud.monitoring.metric.Metric` by providing
+specific values for the available labels. For more information on creating metric descriptors,
+see the `Metric Descriptors`_ section.
+
+You must also create a fully parameterized :class:`~gcloud.monitoring.resource.Resource` from one
+of the available ``ResourceDescriptors`` which can be listed using the
+:class:`~gcloud.monitoring.resource.ResourceDescriptor` class, which similarly requires that its
+available labels have values specified. The ``project-id`` label is a special case that should
+not be specified.
+
+>>> from gcloud import monitoring
+>>> # Create a Resource by its resource name, assign labels as dict
+>>> resource = client.resource('gce_instance', labels={
+...     'instance_id': '1234567890123456789',
+...     'zone': 'us-central1-f'
+... })
+>>> # Create a Metric using its metric descriptor. assigning label as dict
+>>> metric = client.metric(type='custom.googleapis.com/my_metric', labels={
+...      'status': 'successful'
+... })
+
+Please refer to the `Metrics`_ documentation for more information.
+
+With a ``Metric`` and ``Resource`` specified, the :class:`~gcloud.monitoring.client.Client`
+can be used to write :class:`~gcloud.monitoring.timeseries.Point` values.
+
+When writing points, the Python type of the value must match the *value_type* specified
+in the associated :class:`~gcloud.monitoring.metric.MetricDescriptor`. For example, a Python
+float will map to ``ValueType.DOUBLE``.
+
+Stackdriver Monitoring supports several **metric kinds**: *GAUGE*, *CUMULATIVE*, and *DELTA*.
+However, *DELTA* metrics can not be created as custom metrics.
+
+*GAUGE* metrics represent only a single point in time, so only the ``end_time`` should be
+specified::
+
+    >>> client.write_point(metric=metric, resource=resource, 3.14, end_time=end) # API call
+
+By default, end_time defaults to :meth:`~datetime.datetime.utcnow()`, so metrics written for the
+current time can be simplified to::
+
+   >>> client.write_point(metric, resource, 3.14) # API call
+
+For *CUMULATIVE* metrics, the same start time should be re-used repeatedly as more points are
+written to the time series. In this context, the start time is sometimes referred to as the *RESET*
+time. Once the cumulative metric is ready to be reset, the *RESET* time should be reset by
+specifying a new ``start_time``. Please refer to the `TimeSeries`_ documentation for more
```

Comment:
Suggestion:

...is also called the _reset_ time and is typically determined when the application process starts up. Cumulative metrics enable the monitoring system to compute rates of increase across process restarts, which would otherwise show up as huge negative spikes.